### PR TITLE
vagrant: make vagrant file run and general fixups

### PR DESCRIPTION
**Pull request by fredriksvensson**
Fixing diversion to make Xen start in vagrant.

Downloading erlang separately in vagrant. Using either the debian
repository and trusty resulted in errors.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -6,19 +6,19 @@ VAGRANTFILE_API_VERSION = "2"
...
-	sudo sh -c 'echo "deb http://packages.erlang-solutions.com/debian wheezy contrib" > /etc/apt/sources.list.d/erlang.list'
-	wget -c http://packages.erlang-solutions.com/debian/erlang_solutions.asc
-	sudo apt-key add erlang_solutions.asc
-	rm -f erlang_solutions.asc
-	sudo apt-get update
-	rm -f erlang_solutions.asc
-fi
-
-sudo apt-get install -y --force-yes erlang git autoconf gcc-arm-none-eabi make build-essential
+sudo apt-get install -y --force-yes git autoconf gcc-arm-none-eabi make build-essential
```
Comment (EarlGray) [inline Vagrantfile]: @proger why do we install `gcc-arm-none-eabi` here? Did you use it for building RPi images?

Comment (proger) [inline Vagrantfile]: Yes, that's where the Vagrantfile originated from.

Comment (proger) [inline Vagrantfile]: #34

Comment (EarlGray) [inline Vagrantfile]: So, would you mind if I removed this? I guess the main usage for this Vagrantfile is to test Xen.

Comment (proger) [inline Vagrantfile]: I don't.

Comment (fredriksvensson) [inline Vagrantfile]: Should I make the changes and make a new pull request for you?

**Comment by proger** [comment]
LGTM, let's merge?

**Pull request merged**